tests/test_get_routes.py

```diff
@@ -0,0 +1,96 @@
+"""
+Contains the tests regarding get requests to the Fund Store API
+"""
+import json
+
+
+def test_get_rounds(flask_test_client):
```

Comment:
Big challenge for you - write a fixture so that we can use test data for...well...the tests so that we never need to touch production data in unit testing.

Reply:
draw

Reply:
To do when we have more info on how persistent storage will be handle